tck/logs/src/main/java/org/eclipse/microprofile/telemetry/logs/tck/application/JulTest.java

```diff
@@ -49,13 +55,48 @@ public static WebArchive createDeployment() {
 
     private static final Logger julLogger = Logger.getLogger("jul-logger");
 
+    private static final String logFilePath = System.getProperty("log.file.path");
+
+    private static final String JUL_INFO_MESSAGE = "a very distinguishable info message";
+    private static final String JUL_WARN_MESSAGE = "a very distinguishable warning message";
+
+    private static final String SYS_OUT = "SystemOut";
+
     @Test
-    void julTest() throws InterruptedException {
+    void julInfoTest() throws IOException {
+        julLogger.log(Level.INFO, JUL_INFO_MESSAGE);
+        try {
+            Assert.assertTrue(checkMessage(SYS_OUT + ".*" + "INFO" + ".*" + JUL_INFO_MESSAGE));
+        } catch (IOException e) {
+        }
+    }
+
+    @Test
+    void julWarnTest() throws IOException {
+        julLogger.log(Level.WARNING, JUL_INFO_MESSAGE);
+        try {
+            Assert.assertTrue(checkMessage(SYS_OUT + ".*" + "WARN" + ".*" + JUL_INFO_MESSAGE));
+        } catch (IOException e) {
+        }
+    }
 
-        julLogger.info("A JUL log message");
-        julLogger.info("A JUL log message");
-        julLogger.info("A JUL log message");
-        julLogger.info("A JUL log message");
-        Thread.sleep(10000);
+    public boolean checkMessage(String logMessage) throws IOException {
+        try {
+            try {
+                Thread.sleep(5000);
+                BufferedReader reader = new BufferedReader(new FileReader(logFilePath));
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    if (line.contains(logMessage)) {
```

Comment:
String `contains` method doesn't do regex matching -- need to use `matches` instead.